components/__init__.py

```diff
@@ -316,7 +316,7 @@ def define_components(reqs):
                           'mkdir -p "$SPDK_PREFIX/share/spdk"',
                           'cp -r include scripts examples/nvme/fio_plugin ' \
                           '"$SPDK_PREFIX/share/spdk"'],
-                libs=['spdk'],
+                libs=['rte_bus_pci'],
```

Comment:
should we check all libraries we use?

Reply:
The problem is that checking will fail because most libraries will report missing symbols. I used 'ldd' to check the individual spdk lib, don't see why it's dependent libs (other spdk libs) are now showed.

Libs with 'rte' prefix looks normal to me, 'ldd' will show all it's dependent libs, so that existence check can pass for it.